<!DOCTYPE html>
<html lang="en-us">

  <head>
  

    <meta charset="UTF-8">
    <title>Flavivirid-GLUE by giffordlabcvr</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="../assets/stylesheets/normalize.css" media="screen">
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="../assets/stylesheets/stylesheet.css" media="screen">
    <link rel="stylesheet" type="text/css" href="../assets/stylesheets/github-light.css" media="screen">

  </head>

  <body>
    <section class="page-header">
      <h1 class="project-name">Flavivirus-GLUE</h1>
      <h2 class="project-tagline">Resources for comparative genomic analysis of flavivirids  (family <i>Flaviviridae</i>)</h2>

      <a href="../../index.html" class="btn">Home</a>
      <a href="./viruses.html" class="btn">Viruses</a>
      <a href="./news.html" class="btn">News</a>
      <a href="https://github.com/giffordlabcvr/Flavivirus-GLUE/zipball/master" class="btn">Download</a>
      <a target="_blank" href="https://github.com/giffordlabcvr/Flavivirus-GLUE/tree/master" class="btn">GitHub</a>
    </section>

    <section class="main-content">

	   <h3>
		 <a id="Background" class="anchor" href="#Background" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Endogenous flavivirid (EFV) data</strong>
	   </h3>
	   <hr>

		<p>
          We have used GLUE to organise the '<b>genomic fossil record</b>' of flavivirids.
          This page provides a description of Flavivirid-GLUE's paleovirus component, and
          quick links to specific data items.
	   </p>
	   

	   <p>
         <b>Please note</b>: links to files on GitHub are mainly designed to indicate where
         these files are located within the repository. 
         To investigate files (e.g. tree files) in the appropriate software context we recommend
         <a href="https://github.com/giffordlabcvr/Flavivirus-GLUE/zipball/master"><b>downloading the entire repository</b></a>
         and browsing locally. 
          
	   </p>

       <br>
	   <h3>
		 <a id="dataProvenance" class="anchor" href="#dataProvenance" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>How were the EFV data generated?</strong>
	   </h3>
       <hr>
 

	   <p>

		 EFV sequences were recovered from whole genome sequence (WGS) assemblies 
		 via <b>database-integrated genome screening (DIGS)</b> using the 
		  <a target="_blank" href="https://giffordlabcvr.github.io/DIGS-tool/"><b>DIGS tool</b></a>.
	   </p>
	   


	   <p>
		 All data pertaining to this screen are included in this repository.
	   </p>

  
	   <ul>

	   <li>
		 The complete list of <b>vertebrate</b> genomes screened can be found
         <a href="https://github.com/giffordlabcvr/Flavivirus-GLUE/blob/master/digs/vertebrate-genomes-screened.csv"><b>here</b></a>.
	   </li>

 
	   <li>
		 The complete list of <b>invertebrate</b> genomes screened can be found
         <a href="https://github.com/giffordlabcvr/Flavivirus-GLUE/blob/master/digs/invertebrate-genomes-screened.csv"><b>here</b></a>.
	   </li>
 
	   <li>
		 The set of <b>flavivirus</b> polypeptide sequences used as probes can be found
         <a href="https://github.com/giffordlabcvr/Flavivirus-GLUE/blob/master/digs/flavi-probes.faa"><b>here</b></a>.
	   </li>
  
	   <li>
		 The final set of <b>flavivirus</b> and <b>EFV</b> polypeptide sequences used as
		 references can be found
         <a href="https://github.com/giffordlabcvr/Flavivirus-GLUE/blob/master/digs/flavi-references.faa"><b>here</b></a>.
	   </li>

	   <li>
		 Input parameters for screening using the
		  <a target="_blank" href="https://giffordlabcvr.github.io/DIGS-tool/"><b>DIGS tool</b></a>
		 can be found
         <a href="https://github.com/giffordlabcvr/Flavivirus-GLUE/blob/master/digs/efv-screen-digs.ctl"><b>here</b></a>.
	   </li>
	   </ul>




	   <br>	   
	   <h3>
		 <a id="Nomenclature" class="anchor" href="#Nomenclature" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Nomenclature for EFVs</strong>
	   </h3>
	   <hr>
	   

		 <p>
		   We have applied a systematic approach to naming EFV, following a convention 
		   developed for <a target="_blank" href="https://retrovirology.biomedcentral.com/articles/10.1186/s12977-018-0442-1">endogenous retroviruses</a>.
		   Each element was assigned a unique identifier (ID) constructed from a defined
		   set of components.
		 </p>

 
		 <p><img src="../assets/images/efv-nomenclature.png" alt="EFV Nomenclature" /></p>

		
	

		 <p>
		   The first component is the classifier ‘EFV’ (endogenous flavivirid). 
		 </p>

		 <p>
		   The second component is a composite of two distinct subcomponents separated by a period:
		  
		   (i)  the name of EFV group;
		   (ii) a numeric ID that uniquely identifies the insertion.
		   The numeric ID is an integer that identifies a unique insertion locus that arose as a
		   consequence of an initial germline infection. Thus, orthologous copies in different
		   species are given the same number. 
		 </p>
		
		 <p>
		   The third component of the ID defines the set of host species in which the ortholog occurs.
		 </p>




	   <br>
	   <h3>
		 <a id="schemaExtensions" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Paleovirus-specific schema extensions</strong>
	   </h3>
	   <hr>


		 <p>
		   The paleovirus component of Flavivirid-GLUE extends GLUE's
		   <a target="_blank" href="http://glue-tools.cvr.gla.ac.uk/#/coreSchema"><b>core schema</b></a>
		   to allow the capture of EFV-specific data.
		   These schema extensions are defined in 
           <a href="https://github.com/giffordlabcvr/Flavivirus-GLUE/blob/master/glue/build/eve/efvSchemaExtensions.glue"><b>this file</b></a>
		   and comprise two additional table: 'locus_data' and 'refcon_data'.
		   Both tables are linked to the main 'sequence' table via the 'sequenceID' field.
  		</p>


 		<p>
		   The '<b>locus_data</b>' table contains EFV locus information: e.g. species, assembly, scaffold, location coordinates.
		</p>

 		<p>
		   The '<b>refcon_data</b>' table contains summary information for individual
		   EFV insertions. It refers to the reference sequences constructed to represent
		   each insertion, which reflect our best efforts to reconstruct progenitor virus
		   sequences as they might have looked when they initially integrated into 
		   the germline of ancestral species. 
		</p>




	   <br>
	   <h3>
		 <a id="Raw EFV sequences and data" class="anchor" href="#Raw_digs_data" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Raw EFV sequences and data</strong>
	   </h3>
	   <hr>
	   

        <p><img src="../assets/images/eve-header2.png" alt="Species with endogenous flaviviruses" /></p>


<blockquote>
		<p>
		  Some of the species in which we identified novel <b>endogenous flaviviral elements (EFVs)</b>
		  Left to right: 
		  freshwater jellyfish (<i>Craspedacusta sowerbyi</i>),
		  long-horned beetle (<i>Anoplophora glabripennis</i>),
		  tadpole shrimp (<i>Lepidurus arcticus</i>),
		  tube-eye fish (<i>Stylephorus chordatus</i>).
		</p>
</blockquote>

	   <br>


		<p>
          Raw FASTA for EFVs recovered via
          <a target="_blank" href="https://giffordlabcvr.github.io/DIGS-tool/">database-integrated genome screening (DIGS)</a>
          are <a href="https://github.com/giffordlabcvr/Flavivirus-GLUE/tree/master/sources/fasta-digs-eve"><strong>here</strong></a>.
 		</p>



		<p>
          Sequence-associated data in tabular format are <a href="https://github.com/giffordlabcvr/Flavivirus-GLUE/blob/master/tabular/eve/efv-side-data.tsv"><strong>here</strong></a>.
          The tabular files contain information about the genomic locations of EFVs. 
  		</p>



	   <br>

	   <h3>
		 <a id="EFV reference sequences and data" class="anchor" href="#Ref_ecv_data" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>EFV reference sequences and data</strong>
	   </h3>

	   <hr>


		 <p>
			We constructed reference sequences for EFVs using alignments of EFV
			sequences derived from the same initial germline colonisation event - i.e.
			orthologous elements in distinct species, and paralogous 
			elements that have arisen via intragenomic duplication of EFV sequences.
		 </p>



		<p>
          EFV consensus/reference FASTA is
          <a href="https://github.com/giffordlabcvr/Flavivirus-GLUE/tree/master/sources/fasta-refseqs-eve"><strong>here</strong></a>.
 		</p>



		<p>
          Tabular formatted metadata for EFV reference sequences is
          <a href="https://github.com/giffordlabcvr/Flavivirus-GLUE/blob/master/tabular/eve/efv-refseq-side-data.tsv"><strong>here</strong></a>.
  		</p>



	   <br>

	   <h3>
		 <a id="Phylogenetic trees" class="anchor" href="#PhylogeneticTrees" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Phylogenetic trees</strong>
	   </h3>
	   <hr>


		<p>

          We used GLUE to implement an automated process for deriving midpoint rooted,
          annotated trees from the alignments included in our project.
 		</p>


		<p>
          Trees were constructed at distinct taxonomic levels:
        
		  <ol>
			<li> <a target="_blank" href="https://github.com/giffordlabcvr/Flavivirus-GLUE/tree/master/trees/major-lineage"><strong>Major lineage-level</strong></a> phylogenies</li>
			<li> <a target="_blank" href="https://github.com/giffordlabcvr/Flavivirus-GLUE/tree/master/trees/minor-lineage"><strong>Minor lineage-level</strong></a> phylogenies</li>
			<li> <a target="_blank" href="https://github.com/giffordlabcvr/Flavivirus-GLUE/tree/master/trees/genus"><strong>Genus-level</strong></a> phylogenies</li>
			<li> <a target="_blank" href="https://github.com/giffordlabcvr/Flavivirus-GLUE/tree/master/trees/subgenus"><strong>Subgenus-level</strong></a> phylogenies</li>
		  </ol>

 		</p>

 

	   <br>
	   <h3>
		 <a id="Publications" class="anchor" href="#Publications" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Related Publications</strong>
	   </h3>
	   <hr>


		<p>
		
		  Bamford CGG, de Souza WM, Parry R and RJ Gifford
		  <b>(2021)</b>
		  <br>
		  Comparative analysis of genome-encoded viral sequences reveals the evolutionary history of the <i>Flaviviridae</i>.
		  <br>
		  preprint <a href="https://www.biorxiv.org/content/10.1101/2021.09.19.460981v1" target="_blank">[view]</a> 
		  <br>
		  <br>

		  Singer JB, Thomson EC, McLauchlan J, Hughes J, and RJ Gifford
		  <b>(2018)</b>
		  <br>
		  GLUE: A flexible software system for virus sequence data.
		  <br>
		  <i>BMC Bioinformatics</i>
		  <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-018-2459-9" target="_blank">[view]</a> 
		  <br>
		  <br>

		  Zhu H, Dennis T, Hughes J, and RJ Gifford
		  <b>(2018)</b>
		  <br>
		  Database-integrated genome screening (DIGS): exploring genomes heuristically using sequence similarity search tools and a relational database.
		  <a href="https://www.biorxiv.org/content/early/2018/04/25/246835" target="_blank">[preprint]</a> 
		  <br>
		  <br>

		  Gifford RJ, Blomberg B, Coffin JM, Fan H, Heidmann T, Mayer J, Stoye J, Tristem M, and WE Johnson
		  <b>(2018)</b>
		  <br>
		  Nomenclature for endogenous retrovirus (ERV) loci.
		  <br>
		  <i>Retrovirology</i>
		  <a href="https://retrovirology.biomedcentral.com/articles/10.1186/s12977-018-0442-1" target="_blank">[view]</a> 
		  <br>
		  <br>

 		</p>


	 <footer class="site-footer">
	   <span class="site-footer-owner"><a href="https://github.com/giffordlabcvr/Flavivirus-GLUE/">Flavivirid-GLUE</a> is maintained by <a href="https://github.com/giffordlabcvr">giffordlabcvr</a>.</span>
	   <span class="site-footer-credits">This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the <a href="https://github.com/jasonlong/cayman-theme">Cayman theme</a> by <a href="https://twitter.com/jasonlong">Jason Long</a>.</span>
	 </footer>

    </section>

  </body>
  
</html>



