<!DOCTYPE html>
<html lang="en-us">

  <head>


    <meta charset="UTF-8">
    <title>Flavivirid-GLUE by giffordlabcvr</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="../assets/stylesheets/normalize.css" media="screen">
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="../assets/stylesheets/stylesheet.css" media="screen">
    <link rel="stylesheet" type="text/css" href="../assets/stylesheets/github-light.css" media="screen">

  </head>

  <body>
    <section class="page-header">
      <h1 class="project-name">Flavivirid-GLUE</h1>
      <h2 class="project-tagline">Resources for comparative genomic analysis of flavivirids (family <i>Flaviviridae</i>)</h2>

      <a href="../../index.html" class="btn">Home</a>
      <a href="./paleoviruses.html" class="btn">Paleoviruses</a>
      <a href="./news.html" class="btn">News</a>
      <a href="https://github.com/giffordlabcvr/Flaviviridae-GLUE/zipball/master" class="btn">Download</a>

      <a target="_blank" href="https://github.com/giffordlabcvr/Flaviviridae-GLUE/tree/master" class="btn">GitHub</a>
    </section>

    <section class="main-content">


	   <h3>
		 <a id="Background" class="anchor" href="#Background" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Virus data included in Flavivirid-GLUE</strong>
	   </h3>
	   <hr>


	   <p>
		 This page provides background information on the <b>virus-associated</b>
		 data items included in the project  - information about endogenous flaviviral elements (EFVs)
		 can be found <a href="./paleoviruses.html""><b>here</b></a>.
	   </p>


	   <p>
     
         <b>Please note</b>: links to files on GitHub are mainly designed to indicate where
         these files are located within the repository. 
         To investigate files (e.g. tree files) in the appropriate software context we recommend
         <a href="https://github.com/giffordlabcvr/Flavivirus-GLUE/zipball/master"><b>downloading the entire repository</b></a>
         and browsing locally. 
          
	   </p>

	   <p>
	   
		 Those specifically interested in <b>hepatitis C virus (HCV)</b>
		 may want to investigate    
		 <a target="_blank" href="https://github.com/giffordlabcvr/HCV-GLUE"><b>HCV-GLUE</b></a>
		 and
		 <a target="_blank" href="https://github.com/giffordlabcvr/NCBI-HCV-GLUE"><b>NCBI-HCV-GLUE</b></a>.		   

		 These GLUE projects were developed specifically for HCV and incorporate
		 a <b>graphical user interface (GUI)</b>
		 that allows users to browse the underlying GLUE database
		 via 'point-and-click' methods.
          
	   </p>

	   <p>
	   
		 The MRC-University of Glasgow Centre for Virus Research hosts an instance of the
		 <a target="_blank" href="http://hcv-glue.cvr.gla.ac.uk/"><b>GUI version of HCV-GLUE</b></a>.
		   
	   </p>

	   <br>
	   <h3>
		 <a id="genomeFeatures" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Flavivirid genome features</strong>
	   </h3>
	   <hr>

	
		<p>

          Currently, four flavivirid genera are recognised: <i>Pegivirus</i>, <i>Pestivirus</i>, <i>Hepacivirus</i> and <i>Flavivirus</i>.
          These genera contain viruses that have monopartite genomes ~10 kilobases (Kb) in length and encoding
          one or more large polyproteins that are co- and post-translationally cleaved to generate mature virus
          proteins. The structural proteins of the virion - capsid (C), premembrane (prM) and envelope (E) - are
          encoded toward the 5’ end of the genome, while genes encoding non-structural (NS) proteins are located
          further downstream.
          
		</p>


        <p><img src="../assets/images/hcv-genome.png" alt="HCV genome" /></p>
 
<blockquote>
		<p>
		   A schematic representation of the hepatitis C virus (HCV) genome. C=capsid; E=envelope; NS=non-structural; Kb=kilobases
		</p>
         
</blockquote>

	   <br>

			
		
		<p>

          A diverse variety of novel ‘flavivirid-like’ virus species have been described over recent years.
          Most of these newly identified viruses have yet to be incorporated into official taxonomy.
          They exhibit a much greater range of variation in genome structure than is found among
          representatives of officially recognised flavivirid genera, with genome sizes ranging up
          to 20Kb, and one novel group – the Jingmenviruses – comprises viruses with genomes
          that are segmented rather than monopartite.

		</p>
		
		
		<p>
		  We defined a <a target="_blank" href="https://github.com/giffordlabcvr/Flavivirus-GLUE/blob/master/tabular/core/features.tsv"><b>standard set of genome features</b></a>
		  for flavivirids, reflecting current knowledge, and
		  <a target="_blank" href="https://github.com/giffordlabcvr/Flavivirus-GLUE/blob/master/glue/build/core/flaviviridaeFeatures.glue">incorporated this information into Flavivirus-GLUE</a>.
		</p>
		


	   <br>


	   <h3>
		 <a id="sequenceData" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Flavivirus sequences and sequence-associated data</strong>
	   </h3>
       <hr>


	   <p>
		 The sequence data in this project are organised into multiple distinct sources.
		 Each source contains data in either GenBank XML or plain FASTA format.
		 The type of data is indicated by the name of the source (all GenBank XML sources 
		 contain 'ncbi' in the name).
	   </p>
	   

	   <p>
		 GenBank XML files are imported into this project directly from
		 <b><a target="_blank" href="https://www.ncbi.nlm.nih.gov/genbank/">NCBI GenBank</a></b>
		 using
         <a target="_blank" href="https://github.com/giffordlabcvr/Flavivirus-GLUE/blob/master/modules/core/flaviviridaeNcbiReferenceImporter.xml">an appropriately configured</a>
		 version of GLUE's <b>GenBank importer module</b>.
		 The core Flavivirid-GLUE project contains a single NCBI-derived source
		 - <a target="_blank" href="https://github.com/giffordlabcvr/Flavivirus-GLUE/tree/master/sources/ncbi-refseqs"><b>ncbi-refseqs</b></a> -
		 that contains 'master reference' genome sequences for each flavivirid species included in this project.		 		 
	   </p>



	   <p>

		 Where possible, we prefer to use sequences obtained via GenBank since it
		 represents the principle source of published nucleotide sequence data.
		 However, FASTA sources can also be used in GLUE,
		 making it straightforward to expand private instances of this GLUE project with 
		 unpublished sequences. 
	   </p>


	   <p>
		 Genbank sequences are uniquely identified within GLUE projects by their
		 GenBank accession numbers. 
         Sequences included in this project are linked to
         <a target="_blank" href="https://github.com/giffordlabcvr/Flavivirus-GLUE/blob/master/tabular/core/flavi-ncbi-refseqs-side-data.tsv"><b>auxiliary data in tabular format</b></a>.
	   </p>
	   <p>
		 Flavivirid sequence-associated data recorded in Flavivirid-GLUE are as follows:
	   </p>


       <br>

	   
	   <table>
	
		   <thead>
			   <tr>
				   <th>Parameter</th>
				   <th>Type</th>
				   <th>Definition</th>
			   </tr>
		   </thead>
	
		   <tbody>

			   <tr>
				   <td><i>full_name</i></td>
				   <td>VARCHAR</td>
				   <td>Full name of the virus this sequence is derived from</td>
			   </tr>

			   <tr>
				   <td><i>name</i></td>
				   <td>VARCHAR</td>
				   <td>Abbreviated name of the virus this sequence is derived from</td>
			   </tr>

			   <tr>
				   <td><i>subfamily</i></td>
				   <td>VARCHAR</td>
				   <td>Taxonomy - virus subfamily (proposed)</td>
			   </tr>

			   <tr>
				   <td><i>supergenus</i></td>
				   <td>VARCHAR</td>
				   <td>Taxonomy - virus supergenus (proposed)</td>
			   </tr>

			   <tr>
				   <td><i>genus</i></td>
				   <td>VARCHAR</td>
				   <td>Taxonomy - virus genus</td>
			   </tr>

			   <tr>
				   <td><i>subgenus</i></td>
				   <td>VARCHAR</td>				   
				   <td>Taxonomy - virus subgenus</td>
			   </tr>

			   <tr>
				   <td><i>clade</i></td>
				   <td>VARCHAR</td>
				   <td>Taxonomy - virus clade</td>
			   </tr>

			   <tr>
				   <td><i>segment</i></td>
				   <td>INTEGER</td>
				   <td>Genome segment (Jingmenviruses only)</td>
			   </tr>

			   <tr>
				   <td><i>host_group</i></td>
				   <td>VARCHAR</td>
				   <td>Taxonomic group of host species</td>
			   </tr>

			   <tr>
				   <td><i>vector_group</i></td>
				   <td>VARCHAR</td>
				   <td>Taxonomic group of vector species</td>
			   </tr>

			   <tr>
				   <td><i>reservoir_group</i></td>
				   <td>VARCHAR</td>
				   <td>Taxonomic group of reservoir species</td>
			   </tr>

			   <tr>
				   <td><i>isolate_name</i></td>
				   <td>VARCHAR</td>
				   <td>Name of the virus isolate this sequence is derived from</td>
			   </tr>

			   <tr>
				   <td><i>isolation_host</i></td>
				   <td>VARCHAR</td>
				   <td>Species (Latin binomial) virus was isolated from</td>
			   </tr>

			   <tr>
				   <td><i>length</i></td>
				   <td>INTEGER</td>
				   <td>Length of the sequence</td>
			   </tr>

			   <tr>
				   <td><i>pubmed_id</i></td>
				   <td>INTEGER</td>
				   <td>PubMed ID of manuscript associated with sequence</td>
			   </tr>

			   <tr>
				   <td><i>gb_create_date</i></td>
				   <td>GenBank</td>
				   <td>GenBank creation date of the sequence</td>
			   </tr>

			   <tr>
				   <td><i>gb_update_date</i></td>
				   <td>VARCHAR</td>
				   <td>Date of most recent GenBank update</td>
			   </tr>

			   <tr>
				   <td><i>country</i></td>
				   <td>VARCHAR</td>
				   <td>Country where virus was isolated</td>
			   </tr>

			   <tr>
				   <td><i>place_sampled</i></td>
				   <td>VARCHAR</td>
				   <td>Location of sampling (state, region, or city)</td>
			   </tr>

			   <tr>
				   <td><i>collection_year</i></td>
				   <td>INTEGER</td>
				   <td>Year virus was isolated</td>
			   </tr>

			   <tr>
				   <td><i>collection_month</i></td>
				   <td>VARCHAR</td>
				   <td>Month virus was isolated</td>
			   </tr>

			   <tr>
				   <td><i>collection_month_day</i></td>
				   <td>VARCHAR</td>
				   <td>Day of month virus was isolated</td>
			   </tr>


		   </tbody>
		
	   </table>

	   <br>
	   <p>


		 These data are recorded in GLUE's underlying relational database. GLUE's
		 <a target="_blank" href="http://glue-tools.cvr.gla.ac.uk/#/coreSchema"><b>core database schema</b></a>
		 was extended to include these fields, as defined in this
		 <a target="_blank" href="https://github.com/giffordlabcvr/Flavivirus-GLUE/blob/master/glue/build/core/flaviviridaeSchemaExtensions.glue"><b>schema extensions file</b></a>.
	  
	   </p>



	   <br>
	   <h3>
		 <a id="virusReferenceSequences" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong><i>Flaviviridae</i> reference genomes - fully annotated sequences</strong>
	   </h3>
	   <hr>

		<p>

 		  We defined 'master' reference sequences to represent recognised flavivirid genera/subgenera, as follows:

	   <br>
	   <h4>
		 <a id="Genus Flavivirus" class="anchor" href="#Epiflavivirinae" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Genus <i>Flavivirus</i></strong>
	   </h4>


		  <ul>
		    <li><b><i>Mosquito-borne flavivirus group 2</i></b>: Yellow fever virus 1 (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_002031"><b>NC_002031</b></a>)</li>
		    <li><b><i>Mosquito-borne flavivirus group 1</i></b>: Dengue virus 1 (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_001477"><b>NC_001477</b></a>)</li>
		    <li><b><i>Tick-borne flaviviruses</i></b>: Powassan virus (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_003687"><b>NC_003687</b></a>)</li>
		    <li><b><i>No-known vector group 1</i></b>: Apoi virus (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_003676"><b>NC_003676</b></a>)</li>
		    <li><b><i>No-known vector group 2</i></b>: Sokuluk virus (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_026624"><b>NC_026624</b></a>)</li>
		    <li><b><i>Dual-host insect-specific flavivirus group</i></b>: Lammi virus (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_024806"><b>NC_024806</b></a>)</li>
		    <li><b><i>Mpulungu flavivirus group</i></b>: Mpulungu flavivirus (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/LC582740"><b>LC582740</b></a>)</li>
		    <li><b><i>Classical insect-specific flavivirus group</i></b>: Kamiti river virus (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_005064"><b>NC_005064</b></a>)</li>
		    <li><b><i>Crustacean flavivirus group</i></b>: Crangon crangon flavivirus (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/MK473878"><b>MK473878</b></a>)</li>

		  </ul>

	   <h4>
		 <a id="Other Flaviviridae genera" class="anchor" href="#Epiflavivirinae" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Other <i>Flaviviridae</i> genera</strong>
	   </h4>


		  <ul>
		    <li><b><i>Tamanavirus</i></b>: Tamana bat virus (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_003996"><b>NC_003996</b>)</a></li>
		    <li><b><i>Jingmenvirus</i></b>: Jingmen tick virus segment 1 (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_024113"><b>NC_024113</b>)</a></li>
		    <li><b><i>Jingmenvirus</i></b>: Jingmen tick virus segment 3 (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_024114"><b>NC_024114</b>)</a></li>
		    <li><b><i>Pestivirus</i></b>: Bovine viral diarrhea virus 1 (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_001461"><b>NC_001461</b>)</a></li>
		    <li><b><i>Pesti-like 1 (PL2)</i></b>: Soybean cyst nematode virus 5 (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_024077"><b>NC_024077</b>)</a></li>
		    <li><b><i>Pesti-like 2 (PL2)</i></b>: Shuangao lacewing virus 2 (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_028373"><b>NC_028373</b>)</a></li>
		    <li><b><i>Hepacivirus</i></b>: Hepatitis C virus (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_004102"><b>NC_004102</b>)</a></li>
		    <li><b><i>Pegivirus</i></b>: Human pegivirus 2 (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_027998"><b>NC_027998</b>)</a></li>
		  </ul>


		</p>


		<p>

          We explicitly defined the locations of genome features
          on master reference sequences using GLUE commands 
          (<b>see <a target="_blank" href="https://github.com/giffordlabcvr/Flavivirus-GLUE/blob/master/glue/build/core/flaviviridaeExplicitFeatureLocations.glue">here</b></a>).
		  
 		</p>
        


	   <br>
	   <h3>
		 <a id="Multiple sequence alignments" class="anchor" href="#MultipleSequenceAlignments" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Multiple sequence alignments - maps of sequence homology</strong>
	   </h3>
	   <hr>

		<p>
          <b>Multiple sequence alignments (MSAs)</b> are the basic currency of comparative genomic analysis.
          MSAs constructed in this study are linked together using
          GLUE's <b>constrained MSA tree</b> data structure.
  		</p>

         
 		<p>
          A 'constrained MSA' is an alignment in which the coordinate space is defined by 
          a selected reference sequence. Where alignment members contain insertions relative
          to the reference sequence, the inserted sequences are recorded and stored
          (i.e. sequence data is never deleted).
  		</p>

         
 		<p>
           
          GLUE projects have the option of using a data structure called an <b>alignment tree</b>
          to link constrained MSAs representing different taxonomic levels,
          and we've used this approach in Flavivirus-GLUE.
 		</p>


 	    <br>

        <p><img src="../assets/images/flavi-msa-tree.jpg" alt="Alignment tree concept" /></p>
 		

<blockquote>
		<p>
		  The schematic figure above shows the '<b>alignment tree</b>' data structure 
		  currently implemented in <b>Flavivirus-GLUE</b>.
          For the highest taxonomic levels (i.e. at the root) we aligned only the most
          conserved regions of the genome, whereas for the lower
          taxonomic levels (i.e. within and below genus level) we aligned complete coding
          sequences.

          We used an alignment tree data structure to link these alignments,
          via a set of common reference sequences.
          The root alignment contains reference sequences for major clades,
          whereas all children of the
          root inherit at least one reference from their immediate parent.
          Thus, all alignments are linked to one another via our chosen set of
          master reference sequences.
		</p>
         
</blockquote>


 	    <br>

            
   
		<p>
             
          
          Alignments in the project include:
         
		  
		  <ol>
			<li> A ‘<a target="_blank" href="https://github.com/giffordlabcvr/Flavivirus-GLUE/blob/master/alignments/root/flaviviridae-root.fna"><strong>root</strong></a>’
			alignment (i.e. family-level) constructed to represent homology between the two largest subgroupings in the <i>Flaviviridae</i>.
			
			<li> ‘<a target="_blank" href="https://github.com/giffordlabcvr/Flavivirus-GLUE/tree/master/alignments/internal/major-lineage"><strong>major-lineage</strong></a>’
			alignments constructed to represent proposed homologies between representative members
			of major <i>Flaviviridae</i> lineages.</li>

			<li> ‘<a target="_blank" href="https://github.com/giffordlabcvr/Flavivirus-GLUE/tree/master/alignments/internal/minor-lineage"><strong>minor-lineage</strong></a>’
			alignments constructed to represent proposed homologies between representative members
			of 'minor' <i>Flaviviridae</i> lineages.</li>

			<li> ‘<a target="_blank" href="https://github.com/giffordlabcvr/Flavivirus-GLUE/tree/master/alignments/internal/genus/"><strong>genus-level</strong></a>’
			alignments constructed to represent proposed homologies between the genomes of
			representative members of specific flavivirid genera.</li>
			
			<li> ‘<a target="_blank" href="https://github.com/giffordlabcvr/Flavivirus-GLUE/tree/master/alignments/internal/subgenus/"><strong>subgenus-level</strong></a>’
			alignments constructed to represent proposed homologies between the genomes of
			representative members of specific flavivirid subgenera.</li>
		  </ol>
         
 		</p>



	   <br>
	   <h3>
		 <a id="Phylogenetic trees" class="anchor" href="#PhylogeneticTrees" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Phylogenetic trees - reconstructed evolutionary relationships</strong>
	   </h3>
	   <hr>


		<p>
          We used GLUE to implement an automated process for deriving midpoint rooted,
          annotated trees from the alignments included in our project.
 		</p>


		<p>
          Trees were constructed at distinct taxonomic levels:
          
		  <ol>
			<li> ‘<a target="_blank" href="https://github.com/giffordlabcvr/Flavivirus-GLUE/tree/master/trees/major-lineage"><strong>major-lineage</strong></a>’
			phylogenetic trees showing reconstructed evolutionary relationships between 
			representative members of major flavivirid lineages.</li>

			<li> ‘<a target="_blank" href="https://github.com/giffordlabcvr/Flavivirus-GLUE/tree/master/trees/genus"><strong>minor-lineages</strong></a>’
			phylogenetic trees showing reconstructed evolutionary relationships between 
			representative members of minor flavivirid lineages.</li>

			<li> ‘<a target="_blank" href="https://github.com/giffordlabcvr/Flavivirus-GLUE/tree/master/trees/genus"><strong>genus-level</strong></a>’
			phylogenetic trees showing reconstructed evolutionary relationships between 
			representative members of specific flavivirid genera.</li>
			
			<li> ‘<a target="_blank" href="https://github.com/giffordlabcvr/Flavivirus-GLUE/tree/master/trees/subgenus"><strong>subgenus-level</strong></a>’
			phylogenetic trees showing reconstructed evolutionary relationships between 
			representative members of specific flavivirid subgenera.</li>
		  </ol>

 		</p>





	 <footer class="site-footer">
	   <span class="site-footer-owner"><a href="https://github.com/giffordlabcvr/Flavivirus-GLUE/">Flavivirus-GLUE</a> is maintained by <a href="https://github.com/giffordlabcvr">giffordlabcvr</a>.</span>

	   <span class="site-footer-credits">This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the <a href="https://github.com/jasonlong/cayman-theme">Cayman theme</a> by <a href="https://twitter.com/jasonlong">Jason Long</a>.</span>
	 </footer>

    </section>


  </body>
  
</html>



